src/commands/app/create.ts

```diff
@@ -117,7 +120,51 @@ export default class AppCreate extends Command {
         }
       }
 
-      this.error(`Could not create the app. Please try again.`);
+      this.error(`Error: Unable to Create App
+        Please try the following steps:
+        1. Check your internet connection.
+        2. Ensure you have enough balance.
+        3. Try again later.
+        4. If you still have problems, please contact support by submitting a ticket at https://console.liara.ir/tickets.`);
```

Comment:
Putting a dot at the end of links can be problematic:

```suggestion
        4. If you still have problems, please contact support by submitting a ticket at https://console.liara.ir/tickets`);
```